src/de/grobox/liberario/TransportNetworks.java

```diff
@@ -579,4 +610,12 @@ TransportNetwork getTransportNetwork(String idString) {
 	private String getString(int res) {
 		return context.getString(res);
 	}
+
+	private String region(String name, String flag) {
+		return flag + " " + name;
+	}
+
+	private String description(String desc, String networks) {
```

Comment:
Thanks for fixing. Now it does auto-expand the strings, right? ;)

I had yet another idea (sorry for that :stuck_out_tongue:): you don't actually need the description helper method. This could be another `setDescription()` method of the `TransportNetwork` that takes two arguments. Would be more elegant, but not necessary to do it like this.